types/netvox/button/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "R312A - Wireless Emergency Button",
+  "name": "R312A",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed